routes/api/v1/forecast.js

```diff
@@ -0,0 +1,118 @@
+var express = require('express');
```

Comment:
@plapicola @TyMazey got a nice and long controller function for you. no specific new questions but would be interested if you have any thoughts on refactors/different organization.